### ISSUE TITLE
adTagUrl not working

### ISSUE THREAD

**Issue by YashRaj2003** [TheWidlarzGroup/react-native-video#3184]
# Bug

## Platform
Which player are you experiencing the problem on:
* iOS
* Android
* Windows

## Environment info

Library version: latest version available
![WhatsApp Image 2023-07-27 at 14 30 31](https://github.com/react-native-video/react-native-video/assets/87427206/0196af34-b3ca-4142-9bcb-e8e6232dedc3)

Device: Android Mi 11x and andorid emulator

**Comment by TDUser91** [comment]
Is this a protected content (DRM) ?

**Comment by YashRaj2003** [comment]
> Is this a protected content (DRM) ?

I am not much aware what is drm. I tried with m3u8 source url as well as MP4 url. 
Adtag is not working.

I found in api docs that there is some customisation need to be made on Android for client side insertion.
https://github.com/react-native-video/react-native-video/blob/master/API.md#enable-client-side-ads-insertion

**Comment by YashRaj2003** [comment]
It seems react native video hasn't published new changes on npm .

Fixed it by install alpha 6.0 version.
Working fine

**Comment by akbari6776** [comment]
> It seems react native video hasn't published new changes on npm .
> 
> Fixed it by install alpha 6.0 version. Working fine

I have an Android app, and to be able to use VAST for video streaming, I upgraded both projects to react-native-video@v6.0.0-alpha.6. In the buildscript/ext block, I added RNVUseExoplayerIMA = true. Similarly, for the video component, I set controls to false and added the adTagUrl. Although the adTagUrl link is correct and in one of the projects the VAST loads at the beginning of the stream, in the other project, a blank screen appears and nothing loads.
![2023-08-10 20 44 14](https://github.com/react-native-video/react-native-video/assets/42941890/468ba071-710a-4b78-b077-e255362405b0)
